modules/install-upgrade/partials/minimum-sizing.adoc

```diff
@@ -5,67 +5,51 @@
 |Node Count
 |Note
 
-.2+|DEV
+|SANDBOX (STANDALONE)
+|(combined) Zookeeper, Broker, and Bookie
+|1
+|
+
+.2+|DEVELOPMENT
 |(combined) Zookeeper, Broker, and Bookie
 |3
-|1 node per AZ +
-In the simplest case, Pulsar can run on only 1 server host as a standalone mode.
-|Admin Console, and Heartbeat
+|1 node per AZ
+|Admin Console and Heartbeat
+|1
+|Shared, 1 server host; placed in any rack
+
+.4+|SINGLE REGION PRODUCTION ENVIRONMENT (1 region)
+|Zookeeper
+|3
+|
+|Broker
+|3
+|
+|Bookie
+|3
+|
+|Admin Console and Heartbeat
 |1
 |Shared 1 server host; placed in any rack
 
-.4+|TEST
+.6+|HIGH-AVAILABILITY PRODUCTION ENVIRONMENT (2 region)
```

Comment:
this is not 2 region number

Reply:
btw, I think we should stick with 1-region number because multiple-region number is as simple as multiplying the region number.